data/hai/hai reveal 1 intro.txt

```diff
@@ -693,16 +712,18 @@ mission "Hai Leaks Response 3"
 	on offer
 		conversation
 			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
-			`	You get down to business when Sayari inquires how your journey went.`
+			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. Everyone takes their seats at an semicircle-shaped table, and the functionary motions you to the middle seat. A massive holoprojector in front of the table turns on.`
 			choice
 				`	"I don't recall there being nearly so many pirates before."`
 			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
 			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
 			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
 			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`
-			`	Mr. Yang pulls out a datapad. "Well then, I'm afraid I may have some bad news. May I?"`
-			`	He gestures to a display and then takes control when indicated. "One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`
-			`	A series of charts are thrown up on the screen. You're not an expert at data analysis, but the highlighted percentages ranging from 4% to 39% do not look promising. The final graphic is a map of stages of what is labeled as 'artifact penetration'. "Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is anything but the truth, at least not while still holding to the claim of being a representative government," explains Mr. Yang.`
+			`	Mr. Yang interrupts, "Well then, I'm afraid I may have some bad news. May I?`
+			`	"One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures." At his word, the holoprojector shows a series of charts.`
+			choice
+				`	(Examine the charts.)`
```

Comment:
Avoid what I call "empty choices," which are choice branches were you're only given a single action choice that could just as easy be replaced with `(Continue.)`. If this is here because the text at this point is too long, then either try to reduce the text more or make the inclusion of a choice here to break up the text more natural, e.g. by having the player be able to comment on something, even if it all leads to the same place. A few lines up in this same mission there's a choice branch with only a single *dialog* choice for the player, which isn't ideal, but I'd say it's better than a single action choice.

Reply:
This doesn't seem very different to the player watching the monk's video during the Free Worlds campaign, but both can be improved on.

Reply:
I intentionally left that like that because I know in the future I'll be revisiting and redoing most of it anyway.

Reply:
Now the player asks, "How much longer can the Republic keep up their side of the ruse?"

The charts come up automatically when Mr. Yang says the words "bad news."